src/CartItem.js

```diff
@@ -24,24 +25,40 @@ class CartItem extends Component {
         const decreaseQuantity = {
             quantity: this.state.quantity - 1
         }
-        this.setState(decreaseQuantity)   
+        this.setState(decreaseQuantity)
         }
     }
 
     render() {
+
+        let productName = this.props.product.inventory_code
+
         return(
-            <div className="Single-product" id={this.props.product.inventory_code}>
-                <span>{this.props.product.product}</span>
-                <span>${this.props.product.price}</span>
-                <span>
-                    <button onClick={this.minusProduct}><i className="fas fa-minus-square icon"></i></button>
-                    <span> {this.state.quantity} </span> 
-                    <button onClick={this.addProduct}><i className="fas fa-plus-square icon"></i></button>
-                </span>
-            </div>
-        ) 
+
+            <section className="Single-product" id={this.props.product.inventory_code}>
+                <div className="product-img">
+                  <img src={require(`./images/${productName}.png`)} />
+                </div>
+                <div className="product-info">
+                  <h4>{this.props.product.product}</h4>
```

Comment:
If we wind up editing our datasets, it would be worthwhile to change the key that holds the product's name from "product" to "productName" so that we can avoid the "product.product" syntax.